boneyardproj/settings.py

```diff
@@ -22,7 +24,7 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = SECRET_KEY
```

Comment:
What is this change accomplishing?

Reply:
'config' was for using decouple, which I couldn't get to work. Now we are just pulling the environmental key from settings.py file in main_app